presentations/en/2021-micro-acquisition-elevator-pitch.md

```diff
@@ -0,0 +1,103 @@
+<!--markdownlint-disable MD003 MD033 MD041 MD036 -->
+
+<style>
+.reveal section img { background:none; border:none; box-shadow:none; }
+</style>
+
+# Micro-Acquisition Pilot
+
+ESDC IT Strategy Team
+
+Note:
+
+- we are here today to talk to you about our procurement pilot and we will
+- but first we want to talk a little about what we believe
+
+---
+
+Trying to continuously improve while also keeping the lights on is hard
+
+Note:
+
+---
+
+Learning new things is essential to keep up with the fast pace of IT
+
+Note:
+
+(and we know that learning often takes a back seat to operational activities)
+
+---
+
+Sharing code across teams, branches and departments can free up valuable time
+
+Note:
+
+- reduce duplication
+- participate in a community
+- OSS code is more secure
```

Comment:
This can be easily challenged. The current "security vulnerability" aspect deals with the supply chain of code. With proprietary code, the supply chain is believed to be more secure due to the liability on the vendor. 

I'd rephrase this to:
- OSS code is secure. The  same security controls used for proprietary software applies.